tests/CommunityToolkit.Aspire.Hosting.Azure.DataApiBuilder.Tests/ContainerResourceCreationTests.cs

```diff
@@ -49,4 +40,66 @@ public void AddDataAPIBuilderContainerDetailsSetOnResource()
         Assert.True(resource.TryGetLastAnnotation(out EndpointAnnotation? httpEndpointAnnotations));
         Assert.Equal(5000, httpEndpointAnnotations.TargetPort);
     }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_DefaultFile_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // defaults to ./dab-config.json which exists in this test project root
+        builder.AddDataAPIBuilder("dab");
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_PortOnly_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        builder.AddDataAPIBuilder("dab", port: 1234);
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_ValidFile_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // file exists in test project root
+        builder.AddDataAPIBuilder("dab", configFilePaths: "./dab-config.json");
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_ValidFileWithPort_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // file exists in test project root
+        builder.AddDataAPIBuilder("dab", port: 1234, configFilePaths: "./dab-config.json");
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_InvalidFile_ThrowsEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // file does not exist in test project root
+        Assert.Throws<FileNotFoundException>(() => builder.AddDataAPIBuilder("dab", configFilePaths: Guid.NewGuid().ToString()));
+    }
+
+    [Fact]
+    public void AddDataAPIBuilderContainer_ValidFiles_NoEx()
+    {
+        IDistributedApplicationBuilder builder = DistributedApplication.CreateBuilder();
+
+        // both files exist in test project root
+        builder.AddDataAPIBuilder("dab", "./dab-config.json", "./dab-config-2.json");
+    }
```

Comment:
This should assert that there are multiple `ContainerMountAnnotation` annotations on the resource reflecting the config files.

Reply:
I am not sure how to do that. Can you point me to the syntax?

Reply:
Check out https://github.com/CommunityToolkit/Aspire/blob/main/tests/CommunityToolkit.Aspire.Hosting.Azure.StaticWebApps.Tests/ResourceCreationTests.cs#L54-L80

We build the distributed app and then you'd get the DAB resource from the app model and it exposes a method to access the annotations.